crates/autopilot/src/run.rs

```diff
@@ -530,21 +535,35 @@ pub async fn run(args: Arguments) {
         max_winners_per_auction: args.max_winners_per_auction,
         max_solutions_per_solver: args.max_solutions_per_solver,
     };
+    let drivers_futures = args
+        .drivers
+        .into_iter()
+        .map(|driver| async move {
+            match infra::Driver::new(
+                driver.url,
+                driver.name.clone(),
+                driver.fairness_threshold.map(Into::into),
+                driver.submission_account,
+            )
+            .await
+            {
+                Ok(driver) => Some(Arc::new(driver)),
+                Err(_) => None,
```

Comment:
Why do we need to introduce a new error when it is completely ignored on the caller's side? Either an Option should be returned or the error needs to be logged here instead.

Reply:
> that we don't use it now doesn't mean it won't be used.

IMO, then it should only be introduced then. Currently, it is redundant.

Reply:
The function is designed to be used inside the same crate.